src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts

```diff
@@ -220,7 +220,7 @@ export class DashboardWidgetWrapper extends AngularDisposable implements OnInit
 	private updateTheme(theme: IColorTheme): void {
 		const el = <HTMLElement>this._ref.nativeElement;
 		const headerEl: HTMLElement = this.header.nativeElement;
-		let borderColor = theme.getColor(themeColors.SIDE_BAR_BACKGROUND, true);
```

Comment:
what is the 2nd parameter for? did you mean to remove it?

Reply:
Setting it to true doesn't do anything: "@param useDefault specifies if the default color should be used. If not set, the default is used."